wasp-ai/src/client/pages/StatsPage.jsx

```diff
@@ -337,11 +337,11 @@ function getWaitingInQueueDuration(stat, logsByProjectId) {
   return getFormattedDiff(start, end);
 }
 
-function getColorValue(colorName) {
+export function getColorValue(colorName) {
```

Comment:
How is it that these got exported?

If they are used somewhere else, maybe it would make sense to extract them into separate file, because why would they be in StatsPage.jsx?